cli-toolchain/graphql.md

```diff
@@ -560,9 +560,9 @@ There are a few important things to think about when making changes to APIs usin
 
 ### @auth
 
-Authorization is required for applications to interact with your GraphQL API. **API Keys** are best used for public APIs (or parts of your schema which you wish to be public) or prototyping, and you must specify the expiration time before deploying. **IAM** authorization uses [Signature Version 4](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html){:target="_blank"} to make request with policies attached to Roles. OIDC tokens provided by **Amazon Cognito User Pools** can also be used for authorization, and simply enabling this provides a simple access control requiring users to authenticate to be granted top level access to API actions. You can set finer grained access controls using `@auth` on your schema which leverages authorization metadata provided as part of these tokens or set on the database items themselves. 
+Authorization is required for applications to interact with your GraphQL API. **API Keys** are best used for public APIs (or parts of your schema which you wish to be public) or prototyping, and you must specify the expiration time before deploying. **IAM** authorization uses [Signature Version 4](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html){:target="_blank"} to make request with policies attached to Roles. OIDC tokens provided by **Amazon Cognito User Pools** or 3rd party OpenID Connect providers can also be used for authorization, and simply enabling this provides a simple access control requiring users to authenticate to be granted top level access to API actions. You can set finer grained access controls using `@auth` on your schema which leverages authorization metadata provided as part of these tokens or set on the database items themselves.
 
-`@auth` object types that are annotated with `@auth` are protected by a set of authorization rules giving you additional controls than the top level authorization on an API. Currently, `@auth` only supports APIs with Amazon Cognito User Pools enabled. You may use the `@auth` directive on object type definitions and field definitions in your project's schema.
+`@auth` object types that are annotated with `@auth` are protected by a set of authorization rules giving you additional controls than the top level authorization on an API. `@auth` directive supports all the providers that AppSync service supports. You may use the `@auth` directive on object type definitions and field definitions in your project's schema.
```

Comment:
Is this completely accurate? This statement implies that things like owner based auth work for everything which isn't true. Perhaps we just remove it completely?

Reply:
```owner``` is not a provider, but an auth mode, providers are: Cognito, IAM, API Key, OIDC, this was my thinking.

Do you want me to rephrase or extend this somehow, or remove it?

Reply:
Yeah, already removed in PR update.